src/eslint-config-adeira/EslintConfig.flow.js

```diff
@@ -13,4 +13,9 @@ type EslintConfigValues =
 type EslintOverrides = $ReadOnlyArray<{ ... }>;
 
 export type EslintConfigRules = { +[string]: EslintConfigValues, ... };
-export type EslintConfig = { +rules: EslintConfigValues, +overrides: EslintOverrides, ... };
+export type EslintConfig = {
+  +rules: EslintConfigValues,
+  +overrides: EslintOverrides,
+  +plugins: string[],
+  ...
+};
```

Comment:
This `EslintConfig` type could be replaced with the following exact (more precise) variant:

```js
export type EslintConfig = {
  +rules: EslintConfigValues,
  +overrides: EslintOverrides,
  +plugins: $ReadOnlyArray<string>,
  +settings?: { +[string]: $FlowFixMe },
  +globals?: { +[string]: string },
};
```